### PR TITLE
Vite config update

### PR DESCRIPTION
**Pull request by thinknoack**
minor changes to docker-compose and vite.config

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -33,6 +33,8 @@ services:
       - "9992:9992"
     environment:
       - NODE_ENV=development
+    volumes:
+      - ./:/srv:rw,z
```
Comment (nellh) [inline docker-compose.yml]:
```suggestion
```

This should be redundant with the app config later.

File: packages/openneuro-app/vite.config.js [hunk 2]
```diff
@@ -27,6 +27,10 @@ export default defineConfig({
     port: 80,
     host: "0.0.0.0",
     cors: true,
+    watch: {
+      usePolling: true, // Enable polling for file watching
```
Comment (nellh) [inline packages/openneuro-app/vite.config.js]: This has a pretty significant performance cost if it isn't needed for a large project like this. Might be better to use these overrides on macOS and document it for developer setup instead.

In `.env`:
```
CHOKIDAR_USEPOLLING=true
CHOKIDAR_INTERVAL=1000
```

**Review by nellh — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (thinknoack) [reply docker-compose.yml]: this has been removed

File: packages/openneuro-app/vite.config.js (hunk 2 above)
Comment (thinknoack) [reply packages/openneuro-app/vite.config.js]: removed this from vite.config and added polling config to docker-compose and config.env

**Review by nellh — APPROVED** [review]

**Pull request merged**